src/Seo/SeoPage.php

```diff
@@ -184,222 +154,139 @@ public function setMetas(array $metadatas)
     {
         return isset($this->htmlAttributes[$name]);
     }
 
-    /**
-     * @return SeoPageInterface
-     */
     public function setHeadAttributes(array $attributes)
     {
         $this->headAttributes = $attributes;
 
         return $this;
     }
 
-    /**
-     * @param string $name
-     * @param string $value
-     *
-     * @return SeoPageInterface
-     */
     public function addHeadAttribute($name, $value)
     {
         $this->headAttributes[$name] = $value;
 
         return $this;
     }
 
-    /**
-     * @param string $name
-     *
-     * @return $this
-     */
     public function removeHeadAttribute($name)
     {
         unset($this->headAttributes[$name]);
 
         return $this;
     }
 
-    /**
-     * @return array
-     */
     public function getHeadAttributes()
     {
         return $this->headAttributes;
     }
 
-    /**
-     * @param string $name
-     *
-     * @return bool
-     */
     public function hasHeadAttribute($name)
     {
         return isset($this->headAttributes[$name]);
     }
 
-    /**
-     * {@inheritdoc}
-     */
     public function setLinkCanonical($link)
     {
         $this->linkCanonical = $link;
 
         return $this;
     }
 
-    /**
-     * {@inheritdoc}
-     */
     public function getLinkCanonical()
     {
         return $this->linkCanonical;
     }
 
-    /**
-     * {@inheritdoc}
-     */
     public function removeLinkCanonical()
     {
         $this->linkCanonical = '';
     }
 
-    /**
-     * {@inheritdoc}
-     */
     public function setSeparator($separator)
     {
         $this->separator = $separator;
 
         return $this;
     }
 
-    /**
-     * {@inheritdoc}
-     */
     public function setLangAlternates(array $langAlternates)
     {
         $this->langAlternates = $langAlternates;
 
         return $this;
     }
 
-    /**
-     * {@inheritdoc}
-     */
     public function addLangAlternate($href, $hrefLang)
     {
         $this->langAlternates[$href] = $hrefLang;
 
         return $this;
     }
 
-    /**
-     * @param string $href
-     *
-     * @return $this
-     */
     public function removeLangAlternate($href)
     {
         unset($this->langAlternates[$href]);
 
         return $this;
     }
 
-    /**
-     * @param string $href
-     *
-     * @return bool
-     */
     public function hasLangAlternate($href)
     {
         return isset($this->langAlternates[$href]);
     }
 
-    /**
-     * {@inheritdoc}
-     */
     public function getLangAlternates()
     {
         return $this->langAlternates;
     }
 
-    /**
-     * @param $title
-     * @param $link
-     *
-     * @return SeoPageInterface
-     */
     public function addOEmbedLink($title, $link)
     {
         $this->oembedLinks[$title] = $link;
 
         return $this;
     }
 
-    /**
-     * @return array
-     */
     public function getOEmbedLinks()
     {
         return $this->oembedLinks;
     }
 
     /**
      * @param mixed $meta
```

Comment:
```suggestion
     * @param string|array $meta
```
based on the code, looks like it could be defined this way.

Reply:
Sure? What about integer or boolean values?

Reply:
I mean, the code is like this:
```php
    /**
     * @param mixed $meta
     */
    private function normalize($meta): array
    {
        if (\is_string($meta)) {
            return [$meta, []];
        }

        return $meta;
    }
```
with other scalar value it will throw a `TypeError`.